scripts/artifacts.sh

```diff
@@ -1,15 +1,18 @@
-# # Generates the 3 metadata files in `/artifacts` and the polkadot.rs file for integration tests
-#
 # This script is to be run from the root of the repository: `scripts/artifacts.sh`
 #
-# It expects a local polkadot node to be running a JSON-RPC HTTP server at 127.0.0.1:9933
+# It expects a local polkadot node to be running a JSON-RPC HTTP server at 127.0.0.1:9944
 # A local polkadot node can be run via:
 # ```
 # git clone https://github.com/paritytech/polkadot.git
 # cd polkadot 
 # cargo build --release
 # ./target/release/polkadot --dev --tmp
 # ```
+#
+# Instead of this script you can also use the binary crate "scripts/artifacts",
```

Comment:
Is there a good reason to keep both around? I reckon that if we are happy with the rsut crate that we just use that to avoid having two similar things around which may diverge in how they work :)

Reply:
Good point, the bash script can be used for spinning up a local node and generating artifacts from that. But I am not sure who is using it, and if it is needed we can make it a CLI option to the binary crate as well.

Reply:
We only wrote it for our own internal use, so if the rust script does what we need then we can ditch it